alts/src/test/java/io/grpc/alts/internal/AltsAuthContextTest.java

```diff
@@ -36,11 +40,15 @@
   private static final String TEST_PEER_SERVICE_ACCOUNT = "peer@gserviceaccount.com";
   private static final String TEST_RECORD_PROTOCOL = "ALTSRP_GCM_AES128";
 
+  private Map<String, String> testPeerAttributes = new HashMap<String, String>();
+  
+
   private HandshakerResult handshakerResult;
   private RpcProtocolVersions rpcVersions;
 
   @Before
   public void setUp() {
+    testPeerAttributes.put("peer", "attributes");
```

Comment:
How about this: we make the map local, but create two static string like TEST_PEER_ATTRIBUTES_KEY and TEST_PEER_ATTRIBUTES_VALUE.

Reply:
good catch, fixed in update